mlflow/server/js/src/home/components/GetStarted.tsx

```diff
@@ -0,0 +1,82 @@
+import { ArrowRightIcon, Typography, useDesignSystemTheme } from '@databricks/design-system';
+import { FormattedMessage } from 'react-intl';
+import { Link } from '../../common/utils/RoutingUtils';
+import { homeQuickActions } from '../quick-actions';
+import {
+  cardCtaStyles,
+  sectionHeaderStyles,
+  getStartedCardContainerStyles,
+  getStartedCardContentStyles,
+  getStartedCardLinkStyles,
+  getStartedIconWrapperStyles,
+} from './cardStyles';
+
+type QuickAction = typeof homeQuickActions[number];
+
+const GetStartedCard = ({ action }: { action: QuickAction }) => {
+  const { theme } = useDesignSystemTheme();
+  const linkStyles = getStartedCardLinkStyles(theme);
+  const containerStyles = getStartedCardContainerStyles(theme);
+  const contentStyles = getStartedCardContentStyles(theme);
+  const iconWrapperStyles = getStartedIconWrapperStyles(theme);
+
+  const card = (
+    <div css={containerStyles}>
+      <div css={iconWrapperStyles}>
+        <action.icon css={{ width: 20, height: 20 }} />
```

Comment:
this CSS can be removed, it looks like it causes the icons to not be centered

<img width="103" height="76" alt="Screenshot 2025-10-06 at 12 48 22 PM" src="https://github.com/user-attachments/assets/b220c5f9-b1ac-4fb1-9f8e-b51fe5cee1be" />

```suggestion
        <action.icon />
```